web-registry/src/index.tsx

```diff
@@ -44,7 +44,7 @@ ReactDOM.render(
     cacheLocation="localstorage"
   >
     <AuthApolloProvider>
-      <IntercomProvider appId={intercomId} autoBoot>
+      <IntercomProvider appId={intercomId} initializeDelay={500}>
```

Comment:
what does this change do? is it just to avoid intercom using resources at first load?